.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+      - run:
+          name: Install dependencies in virtual environment
+          command: |
+            virtualenv ~/venv
+            source ~/venv/bin/activate
+            pip install -r requirements.txt
+      - save_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+          paths:
+            - "~/venv"
+      - run:
+          name: Run unit tests
+          command: |
+            source ~/venv/bin/activate
+            mkdir test-results
+            python -m pytest -v --junitxml=test-results/junit.xml
+      - store_test_results:
+              path: test-results
+      - store_artifacts:
+              path: test-results
```

Comment:
I didn't quite understand why we are storing artifacts? Looking at the `test-circleci` repo and the results for the CircleCI test [here](https://app.circleci.com/pipelines/github/boolean5/test-circleci/29/workflows/88f18f3d-fd60-4ef8-8229-63a5fb152f98/jobs/29/artifacts) it seems those artifacts, the test results, are broken. Also, CircleCI always stores the build and its output in the pipelines and I don't think we'll need the artifact to store the test results. What do you think?

Reply:
I also noticed that the xml file is not viewable in the browser but it can be downloaded and
inspected. You are right however, storing it is redundant, so I went ahead and removed it.